perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -385,7 +385,8 @@ sub munge_contacts_to_bodies {
     my ($self, $contacts, $report) = @_;
 
     # Make sure Bucks grass cutting reports are routed correctly
-    FixMyStreet::Cobrand::Buckinghamshire::munge_contacts_to_bodies($self, $contacts, $report);
+    my $bucks = FixMyStreet::Cobrand::Buckinghamshire->new;
+    $bucks->munge_contacts_to_bodies($contacts, $report);
```

Comment:
Do we need a check here so this only applies to reports within Bucks somehow?

Reply:
The Bucks `munge_contacts_to_bodies` method returns early if the report doesn't have an extra field called `speed_limit_greater_than_30`, so my thinking was that would be a rare enough occurrence that it's probably safe to leave it like that. Do you think that's enough?

Reply:
Oh and it checks that the report category is "Grass cutting". So it will only perform the WFS lookup if the report category is "Grass cutting" _and_ the report has an extra field called `speed_limit_greater_than_30`.

Reply:
Oh I was looking at the wrong version of the code 🤦 It won't run the query if the category isn't "Grass cutting", but will run on any "Grass cutting" category, so I should probably do something a bit smarter there.

Reply:
It doesn't check the field exists, does it? It not existing is the reason it then does the lookup. There are 3 councils with "Grass cutting", others are Peterborough and Bristol. So I think it would be good to have a check somehow - `munge_report_new_category_list` and `munge_report_new_bodies` in the same file both do a name look up on bodies, which I guess you have here either via the $contacts body or the bodies stash.